src/main/java/com/nique1/musicmanagementsystem/song/SongService.java

```diff
@@ -0,0 +1,11 @@
+package com.nique1.musicmanagementsystem.song;
+
+import java.util.List;
+
+public interface SongService {
```

Comment:
W tej formie ten interfejs jest zbędny, co ciekawe identyczny antywzorzec zastosowała Magda, ciekawe skąd Wam się to wzięło 🤔